.github/workflows/backwards_compatibility_marqo_orchestrator.yml

```diff
@@ -0,0 +1,176 @@
+# Orchestrator workflow
+name: Marqo Compatibility Tests Orchestrator
+
+on:
+  push:
+    branches:
+      - mainline
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+    inputs:
+      to_version:
+        description: 'Target Marqo version'
+        required: true
+      to_image:
+        description: 'Target Marqo image (optional)'
+        required: false
+      max_versions_to_test:
+        description: 'Max versions to test'
+        required: false
+
+# Setting MAX_VERSIONS_TO_TEST, this can be a configurable value or if no input is provided, it can be a default value.
+env:
+  MAX_VERSIONS_TO_TEST: ${{ github.event.inputs.max_versions_to_test || 3 }}
+
+jobs:
+  check-if-image-exists:
+    # Responsible for deciding if we should invoke build_push_img.yml GitHub actions workflow in the same repo.
+    # We do not want to build and push the image if it already exists in the ECR registry, which will be the case if this is a manual developer initiated re-run using the same commit.
+    name: Check if image already exists in ECR
+    runs-on: ubuntu-latest
+    environment: marqo-test-suite
+    outputs:
+      image_exists: ${{ steps.check-image.outputs.image_exists }}
+    steps:
+      - name: Checkout marqo repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.ECR_PUSHER_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ECR_PUSHER_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      # step to check for image existence - it uses aws cli to check if the image exists in the ECR registry "marqo-compatibility-tests"
+      - name: check for image existence
+        id: check-image
+        run: |
+          echo "Checking for image existence"
+          if aws ecr describe-images --repository-name marqo-compatibility-tests --image-ids imageTag=${{ github.sha }}; then
+            echo "::set-output name=image_exists::true"
+            echo "Image already exists in ECR"
+          else
+            echo "::set-output name=image_exists::false"
+            echo "Image doesn't exist" 
+          fi
```

Comment:
set-output is deprecated, change it to $GITHUB_OUTPUT. see https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples for details

Reply:
sure